Data-Science/README.md

```diff
@@ -0,0 +1,148 @@
+
+
+### Mathematics for data science
+
+- [Mathematics and statistics for data science -Youtube(codebasics)](https://www.youtube.com/playlist?list=PLeo1K3hjS3uuKaU2nBDwr6zrSOTzNCs0l)
+- [Maths for data science masterclass -Udemy(Jose Portilla,Krista King)](https://www.udemy.com/share/107qEq/)
+- [Maths for data science -GFG(GeeksForGeeks)](https://www.geeksforgeeks.org/maths-for-data-science/)
+
+### Probabiliity
+
+- [Probability and stastics for data science -Khan Academy](https://www.khanacademy.org/math/statistics-probability/probability-library)
+- [Probability -Probability4datascience(Stanley H. Chan)](https://probability4datascience.com/ch02.html)
+- [Introduction to Probability(Joseph K. Blitzstein, Jessica Hwang)](https://drive.google.com/file/d/15Y0oFNHQRls1qvQNvO3DFLJVhIZvUjTD/view?usp=sharing)
+
+### Statistics
+
+- [Statistics for data science -Youtube(Great Learning)](https://youtu.be/Vfo5le26IhY)
+- [Statistics A full university course -Youtube(freecodeCamp.org)](https://youtu.be/xxpc-HPKN28)
+- [Statistics fundamentals -Youtube(StatQuest with Josh Starmer)](https://youtube.com/playlist?list=PLblh5JKOoLUK0FLuzwntyYI10UQFUhsY9)
+- [Statistics for data science -Youtube(CampusX)](https://youtu.be/DUT4WEUngt0)
+- [Intro to Descriptive Statistics -Udacity](https://www.udacity.com/course/intro-to-descriptive-statistics--ud827)
+- [Fundamentals Of Statistics For Data Scientists -towardsdatascience(Tatev Karen Aslanyan)](https://towardsdatascience.com/fundamentals-of-statistics-for-data-scientists-and-data-analysts-69d93a05aae7)
+
+
+### Scipy
+
+- [Python Scipy -Youtube(Mr. P Solver)](https://youtu.be/jmX4FOUEfgU)
+- [Scipy repository for Scipy -Github(cs231)](https://cs231n.github.io/python-numpy-tutorial/#scipy)
+
+### Data cleaning
+
+- [Data cleaning -Youtube(Alex The Analyst)](https://youtu.be/bDhvCp3_lYw)
+- [Data Cleaning course -datacamp(app.datacamp.com)](https://app.datacamp.com/learn/courses/cleaning-data-in-python)
+- [Data cleaning course -kaggle(kaggle.com)](https://www.kaggle.com/learn/data-cleaning)
+- [Data cleaning documentation -kaggle(Ashish Gaurav)](https://www.kaggle.com/ashishg21/data-cleaning-and-some-analysis-shoe-prices)
+
+### Data Visualization
+
+- [Data Visualization full playlist -Youtube(Corey Schafer)](https://www.youtube.com/watch?v=UO98lJQ3QGI&list=PL-osiE80TeTvipOqomVEeZ1HRrcEvtZB_)
+- [Data Visualization tutorial series -Youtube(sentdex)](https://www.youtube.com/watch?v=q7Bo_J8x_dw&list=PLQVvvaa0QuDfefDfXb9Yf0la1fPDKluPF)
+- [Data Visualization python -Youtube(Intellipaat)](https://www.youtube.com/live/_YWwU-gJI5U?feature=share)
+- [Data Visualization -Youtube(datacamp)](https://app.datacamp.com/learn/courses/introduction-to-data-visualization-with-matplotlib?fbclid=IwAR1OrJSdZ2LVD_c1o3d-_1I7Nhq8OZ3pzTu4010E_XWEmMc0KYsTosz8CIU)
+- [Data Visualization -Kaggle(kaggle)](https://www.kaggle.com/learn/data-visualization)
```

Comment:
Perhaps can make a parent category called Data manipulation for these two?

Reply:
made data manipulation as parent category